### PR TITLE
[WIP] Set up GitHub pages for website

### PR DESCRIPTION
**Pull request by kezhenxu94**
The GitHub pages are previewed locally, it's generally available but need to check all hyperlinks work well, as well as the theme and outlook

### CODE REVIEW

**Comment by wu-sheng** [comment]
Should I merge this?

**Comment by kezhenxu94** [comment]
> Should I merge this?

Please hold on, I'll check again later tonight and I'll ping you when I finish

File: package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "document-cn-translation-of-skywalking",
+  "version": "0.0.1",
+  "description": "SkyWalking 文档社区翻译版",
+  "author": "kezhenxu94 <kezhenxu94@apache.org>",
```
Comment (wu-sheng) [inline package.json]: Please add other Authors, at least GitHub names.

Comment (kezhenxu94) [reply package.json]: Hi @wu-sheng , I thought the author means "the one who create this file" (which is wrong), and I just search the documentation of [npm-package](https://docs.npmjs.com/files/package.json#people-fields-author-contributors) , it says: 

> The “author” is one person. “contributors” is an array of people. A “person” is an object with a “name” field and optionally “url” and “email”, like this:

> npm also sets a top-level “maintainers” field with your npm user info.

Then I'll remove the `author` and add `contributors` instead, thanks for pointing it out

Comment (wu-sheng) [reply package.json]: Thanks.

**Pull request closed without merging**